whelktool/scripts/2020/01/remodel-link-fields/move-linkfield-to-instance.groovy

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
Nice script! Minor thoughts:
* The script does take legacy properties into *some* account. Might update the NOTE to explain that they're temporarily kept to be cleaned up in the following steps.
* `docid` is passed at different positions throughout the functions. Might be prudent to be consequent here (or, *perhaps*, pass a logging Closure if it makes things clearer).

Reply:
Fix it 👍